documents/serializers/document.py

```diff
@@ -18,9 +18,10 @@
 
 
 def status_to_representation(representation):
+    status_timestamp = representation.pop("status_timestamp")
```

Comment:
Why here is this pop?

Reply:
By default the status timestamp would be shown as an individual field in the response, but here it's used as a part of status structure. Pop is used to remove the key and value from the original representation, and its value is moved into status structure.